### ISSUE TITLE
✨ Feature Suggestion | Linux desktop environment section?

### ISSUE THREAD

**Issue by LimeInkling** [privacytools/privacytools.io#2221]
## Description
I've been looking into a few desktop environments for my Linux distro (Arch). I've been particularly interested in KDE Plasma, but I've found that an application, ksysguard (the task manager) depends on libksysguard, which in turn depends on qt5-webengine, which 'provides support for web applications using the Chromium browser project'.

This has made me think about which desktop environments may not be as privacy-friendly as others. Even if it is only a very minor dependency, I still reckon stuff like this should be looked into and a list compiled. What do you think?

**Comment by GintokiHub** [comment]
I think these factors are very interesting (to me at least) and matter to loads of people. I would love there to be a section about it. 
However I've heard repeated ( tell me if I am wrong ) that privacytools.io is meant as a resource primarily for the 'noobs' - between - reasonably tech savvy individual. 
This would perhaps fall under a more advanced topic and material which might not mash with the main theme of the site? 

I am all for perhaps a section somewhere on the site which would open up a portal towards these kinds of topics for those of us and everyone out there which do dive deeper into these topics or would like to and learn about it. 
Should I create a separate issue for this?

**Comment by LimeInkling** [comment]
@GintokiHub I never really got that feel from the site myself - this is the same site I'd heard about Arch Linux from, after all.

The way I see it is; if the site suggests distros which either come with a DE or allow you to choose your own (pretty much all of them, albeit Arch and Alpine being the only ones actually _requiring_ you to) then it would be nice to be warned about the privacy implications of each, and steps you could take to mitigate it.

**Comment by larsmagnusherland** [comment]
I think the Operating System should be split into "Beginner, Novice, Advance" or something similar and add more potential operating systems for them. Going from point-click to fully terminal installation will be horrible for many. Grouping them into levels, you will know at least where to start when you want to become more privacy aware.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
I would like to install Arch but i don't know what desktop environment i need. Any privacy focused recommendations?

**Comment by Pangaea666** [comment]
I lean more towards having one category for "Desktop Linux distros" (or something to that effect), instead of splitting it up into "Introductory" and "Advanced". But there could be a tag/ranking/star system for each that said something about how beginner friendly it is. Ubuntu and Linux Mint (if added), would probably be for "Beginners", while something like Arch is "Advanced". Then instead try to rank the distros on privacy alone, at least if there are tangible differences between them, a bit akin to for browsers.

I'm honestly not sure how the different distros rank privacy-wise, but unless there is something inherently bad about distros like Linux Mint and Manjaro, I reckon it makes sense to mention them, since they are both rather popular distros. At least if going by [distrowatch](https://distrowatch.com/dwres.php?resource=popularity) or [gamingonlinux](https://www.gamingonlinux.com/users/statistics)

I currently use Linux Mint myself, and have "mono-booted" it for years while being a noob, so I can vouch for it being beginner friendly. Have never tried Manjaro so not sure where that would rank.
